src/generator/add-vscode-config/index.js

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const Generator = require('yeoman-generator')
+const fs = require('fs-extra')
+
+/*
+    'initializing',
+    'prompting',
+    'configuring',
+    'default',
+    'writing',
+    'conflicts',
+    'install',
+    'end'
+*/
+
+const Default = {
+  DESTINATION_FILE: '.vscode/launch.json',
+  SKIP_PROMPT: false,
+  SERVER_DEFAULT_PORT: 9080
+}
+
+const Option = {
+  DESTINATION_FILE: 'destination-file',
+  SKIP_PROMPT: 'skip-prompt',
+  SERVER_DEFAULT_PORT: 'server-default-port'
+}
+
+class AddVsCodeConfig extends Generator {
+  constructor (args, opts) {
+    super(args, opts)
+
+    // options are inputs from CLI or yeoman parent generator
+    this.option(Option.DESTINATION_FILE, { type: String, default: Default.DESTINATION_FILE })
+    this.option(Option.SKIP_PROMPT, { type: Boolean, default: Default.SKIP_PROMPT })
+    this.option(Option.SERVER_DEFAULT_PORT, { type: Number, default: Default.SERVER_DEFAULT_PORT })
```

Comment:
not relevant anymore. the port is regex derived from the console log

Reply:
fixed in https://github.com/adobe/aio-cli-plugin-app-dev/pull/14/commits/dacf5648d91e41afeb2a924dc82b8ddaf8103148